gcp/appengine/blog/content/posts/almalinux-and-rocky-linux-join-osv.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "AlmaLinux and Rocky Linux join OSV"
+date: 2023-05-08T16:00:00Z
+draft: false
+author: OSV Team
+---
+
+Two new Linux distributions have been added to the OSV database. With the addition of [AlmaLinux](https://almalinux.org/) and [Rocky Linux](https://rockylinux.org/), the OSV database is now made up of advisories from 18 sources, including language ecosystems and Linux distributions.
+<!--more-->
+
+AlmaLinux and Rocky Linux were both started in response to CentOS being discontinued. These distros are open source, community driven, and 100% compatible with Red Hat Enterprise Linux®. Both maintainer teams worked closely with the OSV team to bring their advisories into the database. We’d like to thank them for their hard work! We know our community will benefit from their data contributions. 
```

Comment:
The CentOS project (now Stream) is working hard to differentiate between old CentOS, now being called "CentOS Linux" vs CentOS Stream.  I'm sure they'd appreciate changing "CentOS" here to read "CentOS Linux" to aid in that.

Reply:
That is a great point. I can definitely change that.

Reply:
What about instead replacing "CentOS being discontinued" with "CentOS moving upstream of RHEL"?  That's more specific and leaves less room for confusion.  I regularly encounter folks that are surprised that the CentOS project still exists because they heard that CentOS was discontinued.